docs/deployment_guide/partner_editable/architecture.adoc

```diff
@@ -20,11 +20,11 @@ As shown in <<architecture1>>, this Quick Start sets up the following:
 *** An Amazon Elastic Compute Cloud (Amazon EC2) Linux instance to bootstrap the installation of VMware TAP in an Amazon EKS cluster and serve as administrator bastion host.
```

Comment:
"TAP" is undefined in the guide, and I don't see "Tanzu Application Platform" abbreviated this way on the VMware website. We can only use "TAP" if VMware authorizes it. If they do, expand the first mention, which appears to be in the **Overview** section. If they don't, then we'll need to spell out every mention that doesn't appear in a URL.